lms/resources/lti_launch.py

```diff
@@ -26,25 +25,6 @@ def __init__(self, request):
         """Return the context resource for an LTI launch request."""
         self._request = request
 
-    @cached_property
```

Comment:
This been always a bit odd, having a property potentially create the course.

Also this whole resource thing has a less clear scope compared to services.